model/simulationtests/autosize_hvac.rb

```diff
@@ -300,6 +300,14 @@ def curve_quadratic(model, c_1constant, c_2x, c_3xPOW2, minx, maxx, miny, maxy)
 cw_loop.addDemandBranchForComponent(chiller)
 swh_loop.addDemandBranchForComponent(chiller) # Heat Recovery
 
+file_name = File.join(File.dirname(__FILE__), 'CoolSys1-Chiller-Detailed.RS0001.a205.cbor')
```

Comment:
I had this comment in the main PR, but where does this file come from? E+ repo? Maybe comment here...

Reply:
it ships with E+, so not a real issue.